pkg/segment/writer/segwriter.go

```diff
@@ -80,6 +83,15 @@ var wipCbufPool = sync.Pool{
 	},
 }
 
+func GetNumOfActiveColWips() int64 {
+	value := atomic.LoadInt64(&activeColWips)
+	return value
+}
+
+func AddNumOfActiveColWips(count int64) {
```

Comment:
these two functions do not need to be exported, they are only used within the writer package

Reply:
addressed in the latest commit.